homeassistant/components/matrix/__init__.py

```diff
@@ -49,10 +51,6 @@
     {
         DOMAIN: vol.Schema(
             {
-                vol.Required(CONF_HOMESERVER): cv.url,
```

Comment:
We should allowing these to be imported into a config entry for easier migration.

Reply:
Do you mean an import from the yaml file or something else?

Reply:
Correct. `async_setup` will still receive the homeserver/ssl/username/password from before we added a config flow. You can start an import flow that imports it so that users don't have to reconfigure the Matrix integration.

See here for an example https://github.com/home-assistant/core/blob/dev/homeassistant/components/ambient_station/__init__.py#L275-L281